packages/core/test/integration/query-interface/show-all-tables.test.ts

```diff
@@ -0,0 +1,169 @@
+import type { PostgresQueryInterface } from '@sequelize/core/src/dialects/postgres/query-interface';
+import { expect } from 'chai';
+import type { Sequelize } from '@sequelize/core';
+import { DataTypes, QueryTypes } from '@sequelize/core';
+import { sequelize, getTestDialect, createSequelizeInstance } from '../support';
+
+const queryInterface = sequelize.queryInterface;
+const dialect = sequelize.dialect;
+const dialectName = getTestDialect();
+
+describe('QueryInterface#showAllTables', () => {
+
+  after(async () => {
+    const _sequelize = createSequelizeInstance();
+    await _sequelize.queryInterface.dropAllTables();
+    if (!dialect.supports.schemas) {
+      return;
+    }
+
+    await Promise.all(['schema_1', 'schema_2', 'schema_3', 'schema_4'].map(async schema => _sequelize.dropSchema(schema)));
+  });
+
+  const getSequelizeInstanceWithSchema = async () => {
+    const _sequelize = createSequelizeInstance({ schema: 'schema_3' });
```

Comment:
We already use underscore prefixes to mute "unused vars" errors, prefer a different name